code/game/objects/items/stacks/medical.dm

```diff
@@ -25,8 +25,8 @@
 		user << "<span class='danger'>\The [M] is dead, you cannot help [t_him]!</span>"
 		return
 
-	if(!istype(M))
-		user << "<span class='danger'>You don't know how to apply \the [src] to [M]...</span>"
+	if(!istype(M, /mob/living/carbon) && !istype(M, /mob/living/simple_animal))
+		user << "<span class='danger'>You don't know how to apply \the [src] to [M]</span>..."

```

Comment:
Make this a period. We use ellipses to indicate an action that takes time to do.